great_expectations/expectations/core/expect_column_values_to_be_in_set.py

```diff
@@ -412,8 +382,9 @@ def _descriptive_example_values_block_renderer(
         return new_block
 
     def validate_configuration(
-        self, configuration: Optional[ExpectationConfiguration]
+        self, configuration: Optional[ExpectationConfiguration] = None
```

Comment:
Does this impact the public API at all?

Reply:
No, the superclass uses a default like this (PyCharm has an error due to the mismatched signature) and the call to `super().validate_configuration(configuration)` does checks for if `None` then use a default at each level of the class heirarchy.